modules/odo-getting-started-code-nodejs.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-nodejs.adoc
+
+:_content-type: Procedure
+[id="odo-getting-started-nodejs_{context}"]
+
+= Generating sample code for Node.js 
+
+Use the https://expressjs.com/[Express] framework to generate the sample code.
+
+.Prerequisites
+* You have installed Node.js and supporting tools
+
+.Procedure 
+
+. Install Express:
++
+[source,terminal]
+----
+npm install express --save
+----
+
+
+. Generate an example project:
++
+.Sample command
+[source, terminal]
+----
+$ npx express-generator
+----
++
+.Sample output
+[source, terminal]
+----
+$ npx express-generator
+  warning: the default view engine will not be jade in future releases
+  warning: use `--view=jade' or `--help' for additional options
+
+
+create : public/
+create : public/javascripts/
+create : public/images/
+create : public/stylesheets/
+create : public/stylesheets/style.css
+create : routes/
+create : routes/index.js
+create : routes/users.js
+create : views/
+create : views/error.jade
+create : views/index.jade
+create : views/layout.jade
+create : app.js
+create : package.json
+create : bin/
+create : bin/www
+
+install dependencies:
+  $ npm install
+
+run the app:
+  $ DEBUG=express:* npm start
```

Comment:
Separate steps?

Reply:
That is in the command output, so not part of the procedure